modules/ipi-install-bmc-addressing.adoc

```diff
@@ -64,3 +64,63 @@ platform:
           password: <password>
           disableCertificateVerification: True
 ----
+[discrete]
+== Redfish APIs
+
```

Comment:
@zaneb @dtantsur We are attempting to document the RedFish apis used by the BM installer, wanted to give you guys a heads up since this covers x86 and arm.

Reply:
Why conflate two completely different changes in one patch? I think you'll have easier time merging ARM first.

Reply:
Generally speaking, we have one PR per section when doing multi-arch enablement work. In hindsight, I suppose we could have had a separate PR for the the redfish api docs.